proc_scripts/load_data.py

```diff
@@ -3,22 +3,60 @@
 import os
 from sympy import symbols
 import logging
-#to use type s = load_data("nameoffile")
-def proc_bruker_deut_IR_withecho_mancyc(s):
-    raise RuntimeError("this is where postprocessing would be implemented -- not implemented yet")
-
-def proc_bruker_deut_IR_mancyc(s, fl=None):
-    logger.info("going through this")
-    s.chunk('indirect',['indirect','ph1','ph2'],[-1,4,2]) #expands the indirect dimension into indirect, ph1, and ph2. inner most dimension is the inner most in the loop in pulse sequence, is the one on the farthest right. Brackets with numbers are the number of phase cycle steps in each one. the number of steps is unknown in 'indirect' and is therefore -1.
-    s.setaxis('ph1',r_[0:4.]/4) #setting values of axis ph1 to line up
-    s.setaxis('ph2',r_[0:2.]/4) #setting values of axis ph1 to line up
+import numpy as np
+fl=figlist_var()
```

Comment:
you cannot / should not be defining a new instance of `figlist_var` here -- this is an issue we've covered before. -- default to `fl=None` below, and generate the instance inside your main script.